### ISSUE TITLE
(PROJECT) Iran Simulations

### ISSUE THREAD

**Issue by kwschultz** [geodynamics/vq#171]
In my VQModels repository, there are two different Iran fault models. One is the Central-Eastern fault system, and the other is the separate Northwestern fault system.   [github.com/kwschultz/vqmodels](url)

Our Iranian collaborator has published only basic earthquake statistics and probabilities for these fault models. [https://gji.oxfordjournals.org/content/206/1/379.full](url)   

To-do:  We can use these models as new places to study fault correlations, or to produce any number of new mini-studies on a fault system other than California.

**Comment by kwschultz** [comment]
You can email Alireza (the Iranian grad student) at     civil.alireza@gmail.com
He is open to collaborations. And you should also ask him for the observed Iran earthquake rates (to put on our frequency-magnitude plots for validation). 

He is also keen to work on coulomb stress change modeling between faults.

It is straightforward to use the QuakeLib library (using it as the python module quakelib), to read in fault models, read in a simulated earthquake slip distribution across a fault, then compute the stress changes on other faults in the model. This type of analysis can be extended to see if our series of simulated earthquakes follows the "triggering" pattern we would expect, where the subsequent earthquakes in the simulation occur where stress increased due to the previous earthquake.

**Comment by babakgab** [comment]
Hi dear professor.
My name is babak.and we( me with my friend,ramin)are working in Tabriz university(iran)  on our thesis about simulating earthquake by using virtual quake software for evaluating recurrence time of large earthquake for seismic hazard assessment .but we have some difficulties using Vq software that we hope overcome by receiving your guidance.
1.	The check results tool at the software does not work properly.we are using most recent version of software 3.1.1(master).
2.	What is the most important tuning parameters for adjusting earthquake rates in the software?
3.	Given the unproper functionality of check results tool,how can we validate our synthethic  catalogs by different way?
4.	Are the parameters such sim.greens.normal_offdiag or sim.greens.normal.diag_min or max must be modified or left unchanged?
With respect
Babak&ramin

**Comment by ccsaylor** [comment]
Hi @babakgab,

I'm fairly new at using VQ myself but will assist as much as I can. To answer some of your questions:

1. Can you elaborate on how the tool is not working properly?

2. The primary tuning parameters of VQ are eta--the dynamic triggering factor--and the stress drop factor. These are described in section 4.3.1 of the VQ user manual available at: https://geodynamics.org/cig/software/vq/

3.

4. All of the input parameters in VQ are described in Appendix A of the VQ user manual. For the ones you listed, the manual says they are only necessary if the mesher is putting your fault elements too close together.

**Comment by babakgab** [comment]
hi.after producing synthetic catalogs how could we examine the various distributions that best describe the PDF of recurrence times?(weibull, brownian passage time,exponential and .....?is this done by vq?or other spftware?
    On Tuesday, August 27, 2019, 11:07:45 PM GMT+4:30, ccsaylor <notifications@github.com> wrote:  
 
 
Hi @babakgab,

I'm fairly new at using VQ myself but will assist as much as I can. To answer some of your questions:
   
   -    
Can you elaborate on how the tool is not working properly?

   -    
The primary tuning parameters of VQ are eta--the dynamic triggering factor--and the stress drop factor. These are described in section 4.3.1 of the VQ user manual available at: https://geodynamics.org/cig/software/vq/

   - 
   -    
All of the input parameters in VQ are described in Appendix A of the VQ user manual. For the ones you listed, the manual says they are only necessary if the mesher is putting your fault elements too close together.

**Comment by ccsaylor** [comment]
Hi again,

VQ includes functionality to plot various statistics about the simulations. Details on how to use PyVQ to do so are included in sections 5.7 and 5.8 of the VQ user manual I linked above.

**Comment by babakgab** [comment]
thanks dear professor
    On Tuesday, September 3, 2019, 10:38:24 PM GMT+4:30, ccsaylor <notifications@github.com> wrote:  
 
 
Hi again,

VQ includes functionality to plot various statistics about the simulations. Details on how to use PyVQ to do so are included in sections 5.7 and 5.8 of the VQ user manual I linked above.

**Comment by babakgab** [comment]
hi again dear professor.i would like to ask a question about talesh fault trace in NW of iran.i downloaded the latest version of  trace file of NW of iran from github and i have problem  with talesh fault trace.accordin to khodaverdian(doi:10.1093/gji/ggw157) the only shortening(thrust) fault in NW of iran is talesh fault and the others are right or left lateral.they extend form earthsurface to  depth of 15 km except talesh fault where the minimum and maximum locking depth are assumed to be 20 and 35 km respectively.i looked at trace file and i found the altitude of -20000 and depth of 15000m for talesh trace file.is it true to think that by this values fault locking depth start at 20 km below earth surfaceand extend to 35 km below earth surface?
    On Tuesday, September 3, 2019, 10:38:24 PM GMT+4:30, ccsaylor <notifications@github.com> wrote:  
 
 
Hi again,

VQ includes functionality to plot various statistics about the simulations. Details on how to use PyVQ to do so are included in sections 5.7 and 5.8 of the VQ user manual I linked above.

**Comment by ccsaylor** [comment]
Yes, that is correct as long as the dip angle of the fault is 0. If not, the fault would extend 15 km along the dip angle from the starting altitude of -20 km.

Apologies for the late reply,



Cameron

**Comment by babakgab** [comment]
hi again dear professor.thanks for your reply.i apologize you for lot of questions.but you are the only man that answers me.thank you.I want to know after producing synthetic catalogues(models) using  various tuning parameter (stress drop-dynamic triggering factor) and given trace files how we can validate these catalogues by knowing the empirical frequency magnitude distribution parameter(double truncated G-R parameters such as beta,maximum earth quake magnitude and lambda(m0)).is there a specific command for that or another way?another question is that how we can choose the earthquakes related to specific fault among simulated earthquakes for studying the statistics of a specific fault?the last question is that do you know kasy w. schultz and how I can ask  some of my question from him? .he has not been active in github from 2016 and he doesnot answers me in researchgate and email.
thank you for your help.sincerely yours.babak


    On Monday, September 23, 2019, 08:58:06 PM GMT+3:30, ccsaylor <notifications@github.com> wrote:  
 
 
Yes, that is correct as long as the dip angle of the fault is 0. If not, the fault would extend 15 km along the dip angle from the starting altitude of -20 km.

Apologies for the late reply,

Cameron

**Comment by ccsaylor** [comment]
I'm not sure that there is a tool that does what you seek since I am not super familiar with everything that VQ can do. I do know however that you can make plots for specific simulation elements--there are examples of how to do so in section 5.7 of the user manual. Kasey Schultz previously worked with VQ but I'm not sure that he is involved with it any more. I wouldn't have a better way of contacting him than you've already tried.



Cameron

**Comment by babakgab** [comment]
thanks.
    On Tuesday, October 8, 2019, 08:47:00 PM GMT+3:30, ccsaylor <notifications@github.com> wrote:  
 
 
I'm not sure that there is a tool that does what you seek since I am not super familiar with everything that VQ can do. I do know however that you can make plots for specific simulation elements--there are examples of how to do so in section 5.7 of the user manual. Kasey Schultz previously worked with VQ but I'm not sure that he is involved with it any more. I wouldn't have a better way of contacting him than you've already tried.

Cameron
